tests/integration/fileTests.py

```diff
@@ -33,6 +34,7 @@ def setUpClass(cls):
         with createApp().app_context():
             # get the submission test user
             sess = GlobalDB.db().session
+            cls.session = sess
```

Comment:
Aaah, just looked at this again. I see what you did there! Thanks for the context.